### PR TITLE
Modify the naming of some languages

### PR DESCRIPTION
**Pull request by XYZscratcher**
Modify the naming of some languages based on https://github.com/leachim6/hello-world/blob/main/contributing.md#naming

### CODE REVIEW

**Comment by MrBrain295** [comment]
You will also need to rename the individual files.

**Review by MrBrain295 — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -497,7 +497,7 @@ Make sure to see [contributing.md](/contributing.md) for instructions on contrib
 * [Kitten](k/Kitten.ktn)
 * [Kivy](k/Kivy.py)
 * [Koa](k/Koa.js)
-* [Kobeři-C](k/Kobe%C5%99i-C.koberice)
+* [Kobeři C](k/Kobe%C5%99i%20C.koberice)
```
Comment (Richienb) [inline readme.md:500]: The renamings should be rechecked. For example, [Kobeři-C](https://github.com/fpeterek/Koberi-C) contains the dash in the name.

Comment (XYZscratcher) [reply readme.md]: Thanks for the reminder, edited

**Comment by MrBrain295** [comment]
@XYZscratcher I'm closing this for now ping me if you want it reopened.

**Pull request closed without merging**